### ISSUE TITLE
Idea capture: Automation of noise pollution

### ISSUE THREAD

**Issue by SirPolaris** [Citizen-Group/Open311#104]
It would be easy to install a sensor that only trips after a certain threshold of noise is produced. 

This can be logged via the reporting tool, and could automate a bylaw response if sustained or frequent.

**Comment by Zebreu** [comment]
Would it not be easier to do by just hooking in a smartphone's microphone instead of using a dedicated sensor?

**Comment by SirPolaris** [comment]
I'm thinking more like a low energy deployed sensor. People in high complaint/troubled areas can have sensors deployed and 24/7 reporting then would flood in. We could work the data and provide bylaw calls automatically or just generate a report for policing. 

![sketch 9](https://user-images.githubusercontent.com/7384146/28253839-a06e5762-6a75-11e7-877e-fba503d878cf.png)

This was just a random thought, I wanted to capture it could be expanded later.

**Comment by Zebreu** [comment]
I see! This could also be great for other issues by the way, there's lot of info that can be garnered from sound.

**Comment by SirPolaris** [comment]
Agreed and the sound doesn't need to be processed either in this case.
Just a threshold trip.
This system supports all kinds of reporting. It will be interesting to develop out novel tools..
